### PR TITLE
Use spread operator instead of Object.assign call

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
**What kind of change does this PR introduce?**
Code refactor

**What is the current behavior?**
- Use `Object.assign` call

**What is the new behavior?**
- Use spread operator

**Checklist**:

- [ ] Documentation N/A
- [x] Tests
- [x] Ready to be merged 
- [ ] Added myself to contributors table  N/A

### CODE REVIEW

**Comment by SaraVieira** [comment]
Hey!

Can you please revert the changes inside the overmind folder? We have christian doing a complete refactor of that part :(

That folder exists for the migartion to overmind

**Comment by MichaelDeBoey** [comment]
On it 🙂

File: packages/app/src/sandbox/eval/loaders/@babel-runtime.no-webpack.js [hunk 1]
```diff
@@ -421,7 +421,7 @@ function M(e) {
     );
   var o = e.descriptor;
   K(e, 'elements', 'An element descriptor');
-  var i = { kind: r, key: t, placement: n, descriptor: Object.assign({}, o) };
```
Comment (SaraVieira) [inline packages/app/src/sandbox/eval/loaders/@babel-runtime.no-webpack.js]: Can you please allso revert this file?

This is generated

Comment (MichaelDeBoey) [reply packages/app/src/sandbox/eval/loaders/@babel-runtime.no-webpack.js]: Reverted 🙂

**Comment by SaraVieira** [comment]
Awesome! Thank you and thank you for the changes :)

**Comment by MichaelDeBoey** [comment]
Always a pleasure! 🙂

**Pull request merged**